glyphs-reader/src/font.rs

```diff
@@ -723,6 +723,28 @@ pub struct Anchor {
     pub pos: Point,
 }
 
+impl Anchor {
+    // an anchor with this name on a mark glyph is used to adjust positions of other anchors
+    const MAGIC_ORIGIN_NAME: &'static str = "*origin";
+
+    /// If this anchor is the magic '*origin' anchor, return its value as a `Vec2`
+    pub fn magic_origin_adjustment(&self) -> Option<Vec2> {
+        if self.name == Self::MAGIC_ORIGIN_NAME {
+            Some(self.pos.to_vec2())
+        } else {
+            None
+        }
+    }
+
+    /// Returns `true` if this anchor has special semantic meaning
+    ///
+    /// Anchors with not be included in IR.
+    pub fn is_special(&self) -> bool {
+        //TODO: figure out what other names we should be ignoring/handling?
+        self.name == Self::MAGIC_ORIGIN_NAME
+    }
```

Comment:
I think we could better encapsulate this. For example, 

```rust
/// If Some, this isn't an anchor itself, rather an adjustment to other anchors
pub fn anchor_adjustment(&self) -> Option<Vec2> {
  // if named origin return our pos as vec2
}
```

source.rs would then just reduce anchors => a single adjustment (if any) and apply it.

Reply:
This is separate because I believe there are going to be additional special anchors that we will need to skip.

Reply:
I think this might feel better if Anchor was an enum then. We already have two distinct cases, Origin and whatever you want to call the kind that actually flow into the output as anchors. We also already have RawAnchor => Anchor so it fits to classify them in that process.